File: app/src/main/java/com/bhm/sdk/demo/entity/DoPostEntity.java
package com.bhm.sdk.demo.entity;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * @author Buhuiming
 */
public class DoPostEntity implements Serializable {

    @SerializedName("code")
    private int code;

    @SerializedName("message")
    private String message;

    @SerializedName("data")
    private DataEntity data;

    public int getCode() {
        return code;
    }

    public String getMessage() {
        return message;
    }

    public DataEntity getData() {
        return data;
    }

    public static final class DataEntity implements Serializable {
        @SerializedName("key")
        private String key;
        @SerializedName("endpoint")
        private String endpoint;
        @SerializedName("params")
        private Object params;

        public String getKey() {
            return key;
        }

        public String getEndpoint() {
            return endpoint;
        }

        public Object getParams() {
            return params;
        }
    }
}

File: app/src/main/java/com/bhm/sdk/demo/entity/DoPostEntity.kt
package com.bhm.sdk.demo.entity

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * @author Buhuiming
 */
@Suppress("unused")
class DoPostEntity : Serializable {
    @SerializedName("code")
    val code = 0

    @SerializedName("message")
    val message: String? = null

    @SerializedName("data")
    val data: DataEntity? = null

    class DataEntity : Serializable {
        @SerializedName("key")
        val key: String? = null

        @SerializedName("endpoint")
        val endpoint: String? = null

        @SerializedName("params")
        val params: Any? = null
    }
}